calico/datamodel_v1.py

```diff
@@ -85,6 +86,10 @@
 ENDPOINT_STATUS_DOWN = "down"
 ENDPOINT_STATUS_ERROR = "error"
 
+# Information intended for use by the DHCP agent.
+DHCP_DIR = ROOT_DIR + "/dhcp" + DHCP_VERSION
+SUBNET_DIR= DHCP_DIR + "/subnet"
```

Comment:
Not right now, but I'm wondering if we could get caught out later.  The only thing that I can see that we're concretely missing is the admin status for the network object.  OpenStack seems to leave everything else up to the subnet object.